#IMPORTANT: the channel range in VRC is different
#			from the channel range in broadcast data.
#
# VRC channel range is [1024-660, 1024+660]
# For Matrice 100: 
#		channle_data[0]: Roll
#		channle_data[1]: pitch 
#		channle_data[2]: throttle 
#		channle_data[3]: yaw
#		channle_data[4]: gear (1684:up, 1324:down)
#		channle_data[6]: mode (1552:P, 1024:A, 496:F)


uint32[16] channel_data
---
bool result
